src/librados/librados_c.cc

```diff
@@ -666,6 +668,135 @@ extern "C" int _rados_inconsistent_pg_list(rados_t cluster, int64_t pool_id,
 }
 LIBRADOS_C_API_BASE_DEFAULT(rados_inconsistent_pg_list);
 
+static void dump_inconsistent(const librados::inconsistent_obj_t& inc,
```

Comment:
i don't think we should use a JSON encoded `inconsistent_obj_t` as part of the interface. probably we can define a C style struct as `inconsistent_obj_t` 's counterpart in C ?

@dzafman what do you think?

Reply:
It makes sense to me.